### ISSUE TITLE
Possibility to have FCC certificate without any real test

### ISSUE THREAD

**Issue by Bam92** [freeCodeCamp/freeCodeCamp#34763]
**Looking forward for reporting a security issue:**
Please report security issues by sending an email to `security@freecodecamp.org` instead of raising a GitHub issue.

**Describe the bug**
I have been testing the FCC platform to see how it works. I started by Responsive Web Design. I did take and pass 2 projects. While the unit test runs well in each project, I was estonished to see that when submitting the solution that app accept any link without any verification i.e you can copy paste the link for project 1 in project 2 input solution and it passes. Also you can even past a non FCC URL and it passes.
In short, one can just come and paste a non-existing URL and get certified in just 5 min or less.

**To Reproduce**
1.Open this [link](https://learn.freecodecamp.org/front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine/).
2. Paste any URL in the solution input
3. Ctrl + Enter
4. And it passes

**Expected behavior**
It would be good to test the submitted solution to insure that it is realy the expected one.

**Comment by Ronkiro** [comment]
Not sure how anyone would track this. It's not really a security concern, but it's hard to make checks for all sites that can host code online.

I was also thinking on that, it's no problem anyone faking those tests, as they can also copy from other replies, and the real objective is learning, not the certificates.

So i submitted an Issue (#35029) that can help one to learn and "solve" this.
I created as a new issue due to being a new implementation, not really a solution to this issue.

**Comment by Bam92** [comment]
You're right, it is not a security issue.

What I propose is to run the same test as in the challenge page in the solution page.

**Comment by Ronkiro** [comment]
> You're right, it is not a security issue.
> 
> What I propose is to run the same test as in the challenge page in the solution page.

The problem is: as the ideas come from your mind, the ways to complete the challenge are infinite. I think there is no way to implement a solution like that, and keep making people thinking out of the box, doing things from zero. Except the suggestion i made to solve it.

**Comment by raisedadead** [comment]
Thanks @Bam92 @Ronkiro 

I am going to close this in favor of #35015, which has gotten more eyes on it now, even though this one was reported earlier. Let's continue the discussion there.
